portfolio/src/main/webapp/sheetsApi.js

```diff
@@ -1,91 +1,27 @@
-const SHEETS_DISCOVERY_DOCS =
-    ['https://sheets.googleapis.com/$discovery/rest?version=v4'];
-
-// Authorization scopes required by the API; multiple scopes can be
-// included, separated by spaces.
-const SCOPE_SHEETS_READ_WRITE = 'https://www.googleapis.com/auth/spreadsheets';
-
 const URL_DOCUMENT_BEGINNING = 'https://docs.google.com/spreadsheets/d/';
 
-// Error codes that the process can throw
-const ERROR_CODE_SHEETS = {
-  POPUP_CLOSED: 'popup_closed_by_user',
-  ACCESS_DENIED: 'access_denied'
-};
-
-/**
- * On load, called to load the auth2 library and API client library.
- */
-function handleClientLoadSheets() {
-  gapi.load('client:auth2', initClientSheets);
-}
-
 /**
  * Initializes the API client library and sets up sign-in state
  * listeners.
  */
 function initClientSheets() {
-  gapi.client
-      .init({
-        apiKey: SHEETS_API_KEY,
-        clientId: SHEETS_CLIENT_ID,
-        discoveryDocs: SHEETS_DISCOVERY_DOCS,
-        scope: SCOPE_SHEETS_READ_WRITE,
-        consent: 'consent'
-      })
-      .then(
-          function() {
-            var sheetsExportButton =
-                document.getElementById('sheets-export-button');
-            var sheetsSignOutButton =
-                document.getElementById('sheets-sign-out-button');
-            var sheetsSignInButton =
-                document.getElementById('sheets-sign-in-button');
-
-            // Listen for sign-in state changes.
-            gapi.auth2.getAuthInstance().isSignedIn.listen(handleUiSheets);
-
-            // Handle the initial sign-in state.
-            handleUiSheets();
-            sheetsExportButton.onclick = handleExportSchedule;
-            sheetsSignOutButton.onclick = handleSignOut;
-            sheetsSignInButton.onclick = handleSignIn;
-          },
-          function(error) {
-            handleError(error);
-          });
-}
-
-/**
- * Called when the sign-in button is clicked. Handles signing in the user and
- * redirecting errors.
- */
-function handleSignIn() {
-  gapi.auth2.getAuthInstance().signIn().catch(function(error) {
-    handleAuthorizationError(error);
-  });
+  var sheetsExportButton = document.getElementById('sheets-export-button');
+  var sheetsSignOutButton = document.getElementById('sheets-sign-out-button');
+  var sheetsSignInButton = document.getElementById('sheets-sign-in-button');
+
+  // Handle the initial sign-in state.
+  handleUiSheets();
+  sheetsExportButton.onclick = handleExportSchedule;
+  sheetsSignOutButton.onclick = handleSignOut;
+  sheetsSignInButton.onclick = handleSignIn;
 }
 
-/**
- * Called when the sign-out button is clicked, handles signing out the user.
- */
-function handleSignOut() {
-  gapi.auth2.getAuthInstance().signOut();
-}
-
-
 /**
  * Called when the sign-in state changes and updates the UI appropriately.
  */
 function handleUiSheets() {
   $('#sheets-export-button').addClass('d-none');
-  if (gapi.auth2.getAuthInstance().isSignedIn.get()) {
+  if (GoogleAuth.isSignedIn.get()) {
```

Comment:
This is just a question. Why did we change some GoogleAuth's to gapi.auth2... and vice versa. Was there no way of making it all GoogleAuth or gapi.auth2?

Reply:
Good catch, I'll change them all to GoogleAuth.

Reply:
Done.